src/core/src/graph_util.cpp

```diff
@@ -344,6 +344,20 @@ void save_model(const std::shared_ptr<const ov::Model>& m, const std::string& ou
     manager.run_passes(cloned);
 }
 
+void save_model(const std::shared_ptr<const ov::Model>& m, const std::wstring& output_model, bool compress_to_fp16) {
+    auto cloned = m->clone();
+    if (compress_to_fp16) {
+        // TODO: Implement on-the-fly compression in pass::Serialize
```

Comment:
do we have a ticket for it?

Reply:
Couldn't find the ticket, so created it: CVS-145380